templates/definition/meter/sungrow-hybrid.yaml

```diff
@@ -208,6 +246,17 @@ render: |
               address: 13050 # Charge/discharge command
               type: writesingle
               decode: uint16
+        # Set max battery discharge power, effectively stops discharging
+        - source: const
+          value: 10 # 0.01 kW
```

Comment:
Why not 0 or a configurable value?

Reply:
Because the value range is from 1 (0.01kW) to X kW, e.g. 1000 for 10kW (depending on model, max ac power of inverter)

Reply:
Value is 10, not 1- why not put 1 then?

Reply:
to keep it in W and keep it easily understandable

Reply:
I adjusted the comment next to it